script.js

```diff
@@ -472,8 +452,13 @@ if (
       const resTime = await fetch(
         `https://lernia-sjj-assignments.vercel.app/api/booking/available-times?date=${inputDate.value}&challenge=3"`
       );
+
+      //removes duplicated time options.
       const dataTime = await resTime.json();
-      dataTime.slots.forEach((slotTime) => {
+      let timeNoDupicated = dataTime.slots;
+      let timeArr = [...new Set(timeNoDupicated)];
```

Comment:
Nice solution! Are you comfortable with the aspects of javascript that are being used here, i.e. the spread operator and `Set()`? They are advanced concepts, but very useful!